x-pack/plugins/infra/common/http_api/metrics_api.ts

```diff
@@ -10,11 +10,16 @@ import * as rt from 'io-ts';
 import { MetricsUIAggregationRT } from '../inventory_models/types';
 import { afterKeyObjectRT } from './metrics_explorer';
 
-export const MetricsAPITimerangeRT = rt.type({
-  from: rt.number,
-  to: rt.number,
-  interval: rt.string,
-});
+export const MetricsAPITimerangeRT = rt.intersection([
```

Comment:
Should be reverted after implementing https://github.com/elastic/kibana/issues/162560